packages/compass-components/src/components/combobox/Combobox.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable filename-rules/match */
```

Comment:
There's something up with linting in compass-components, btw. It complains about this and the leafygreen imports in vscode, but clearly not in `npm run check`. But also doesn't complain about the type imports and a bunch of things below - only in vscode and (possibly) if you run eslint directly in the package. Seems to fail in such a way that it passes.